polaris-tokens/src/themes/base/color.ts

```diff
@@ -144,7 +144,9 @@ export type ColorBorderAlias =
   /** Specialty and component border colors. */
   | 'input-border-active'
   | 'input-border-hover'
-  | 'input-border';
+  | 'input-border'
+  | 'bevel-border'
+  | 'bevel-transparent-border';
```

Comment:
Looking at other `color-<subgroup>` tokens, the naming convention puts `-transparent` after the subgroup name e.g. `color-bg-surface-transparent` and `color-bg-fill-transparent`. Should this be `color-bevel-border-transparent`?